geniza/corpus/templates/admin/corpus/document/change_form.html

```diff
@@ -18,11 +18,21 @@
         </fieldset>
     {% endif %}
     {% if original.has_transcription %}
-        <h2>Transcription</h2>
-        {% for footnote in original.footnotes.all %}
-            {% if footnote.content %}
-                {% include "footnotes/transcription.html" with transcription=footnote.content %}
-            {% endif %}
-        {% endfor %}
+        <fieldset class="module aligned transcriptions-field">
+            <div class="form-row">
+                <label>Transcription(s)</label>
```

Comment:
let's use `pluralize` here

Reply:
I always forget about that! Done in c62ff7a 👍